test/test_holidays.rb

```diff
@@ -149,7 +156,7 @@ def test_sub_regions
   def test_sub_regions_holiday_next
     # Should return Victoria Day.
     holidays = Holidays.next_holidays(2, [:ca], Date.civil(2008,5,1))
-    assert_equal 1, holidays.length
+    assert_equal 2, holidays.length
```

Comment:
I think it didn't pick up the second one because of the dates it was looking for so really this directly related to the issue we are fixing with this PR. 😄